$NetBSD: patch-builds_unix_freetype-config.in,v 1.1 2017/09/18 16:54:06 adam Exp $

Do not substitute %PKG_CONFIG% as it ends with temporary DESTDIR.

--- builds/unix/freetype-config.in.orig	2017-09-18 15:52:06.000000000 +0000
+++ builds/unix/freetype-config.in
@@ -14,7 +14,7 @@ export LC_ALL
 
 
 # if `pkg-config' is available, use values from `freetype2.pc'
-%PKG_CONFIG% --atleast-pkgconfig-version 0.24 >/dev/null 2>&1
+pkg-config --atleast-pkgconfig-version 0.24 >/dev/null 2>&1
 if test $? -eq 0 ; then
   # note that option `--variable' is not affected by the
   # PKG_CONFIG_SYSROOT_DIR environment variable
@@ -23,17 +23,17 @@ if test $? -eq 0 ; then
     export PKG_CONFIG_SYSROOT_DIR
   fi
 
-  prefix=`%PKG_CONFIG% --variable prefix freetype2`
-  exec_prefix=`%PKG_CONFIG% --variable exec_prefix freetype2`
+  prefix=`pkg-config --variable prefix freetype2`
+  exec_prefix=`pkg-config --variable exec_prefix freetype2`
 
-  includedir=`%PKG_CONFIG% --variable includedir freetype2`
-  libdir=`%PKG_CONFIG% --variable libdir freetype2`
+  includedir=`pkg-config --variable includedir freetype2`
+  libdir=`pkg-config --variable libdir freetype2`
 
-  version=`%PKG_CONFIG% --modversion freetype2`
+  version=`pkg-config --modversion freetype2`
 
-  cflags=`%PKG_CONFIG% --cflags freetype2`
-  dynamic_libs=`%PKG_CONFIG% --libs freetype2`
-  static_libs=`%PKG_CONFIG% --static --libs freetype2`
+  cflags=`pkg-config --cflags freetype2`
+  dynamic_libs=`pkg-config --libs freetype2`
+  static_libs=`pkg-config --static --libs freetype2`
 else
   prefix="%prefix%"
   exec_prefix="%exec_prefix%"
